cmd/kube-apiserver/app/options/globalflags.go

```diff
@@ -23,7 +23,6 @@ import (
 
 	// ensure libs have a chance to globally register their flags
 	_ "k8s.io/apiserver/pkg/admission"
-	_ "k8s.io/kubernetes/pkg/cloudprovider/providers"
```

Comment:
Do we have any sort of test to ensure that flags such as "azure-container-registry-config" or "cloud-provider-gce-lb-src-cidrs" are advertised and respected?

Reply:
> Do we have any sort of test to ensure that flags such as "azure-container-registry-config" or "cloud-provider-gce-lb-src-cidrs" are advertised and respected?

I'm only removing "cmd/controller-manager/app/options/globalflags.go" which only registered "cloud-provider-gce-lb-src-cidrs". I moved registration to the main. If that flag isn't bound properly, the main should panic:

https://github.com/kubernetes/kubernetes/blob/f0beaf46db82e713392a5c0f880e27a0e14c0690/staging/src/k8s.io/apiserver/pkg/util/globalflag/globalflags.go#L66